index.bs

```diff
@@ -153,22 +153,31 @@ The following is an informative summary of the format specifiers processed by th
   </tr>
 </table>
 
-<h3 id="printer" aoid="Printer" nothrow>Printer(<var>logLevel</var>, <var>args</var>)</h3>
+<h3 id="printer" aoid="Printer" nothrow>Printer(<var>logLevel</var>, <var>args</var>, <var>options</var>)</h3>
 
-The printer operation is implementation-defined. It accepts a log level indicating severity, and a
-List of arguments to print (which are either JavaScript objects, of any type, or are
-implementation-specific representations of printable things such as a stack trace, a <a>group</a>,
-or objects with either <a>generic JavaScript object formatting</a> or
-<a>optimally useful formatting</a> applied). How the implementation prints <var>args</var> is up to
-the implementation, but implementations should separate the objects by a space or something
-similar, as that has become a developer expectation.
+The printer operation is implementation-defined. It accepts a log level indicating severity, a List
+of arguments to print, and an optional object of implementation-specific formatting options.
+Elements appearing in <var>args</var> will be one of the following:
+
+<emu-alg>
+  1. JavaScript objects of any type.
+  1. Implementation-specific representations of printable things such as a stack trace or
+  <a>group</a>.
+  1. Objects with either <a>generic JavaScript object formatting</a> or
+  <a>optimally useful formatting</a> applied.
+</emu-alg>
+
+Optionally, if the <var>options</var> object is not undefined, implementations can use
+<var>options</var> to apply implementation-specific formatting to the elements in <var>args</var>.
+How the implementation prints <var>args</var> is up to the implementation, but implementations
```

Comment:
I think this is nicer as a second paragraph. Once-sentence paragraphs are reasonably common in specs.